src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -327,6 +329,29 @@ public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context,
             return Text(runner.GetFormattingInfo(), irContext, args, runner.CancellationToken);
         }
 
+        public static FormulaValue TextToBlob(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)

```

Comment:
>TextToBlob [](http://example.com/codeflow?start=35&length=10)

Can we rename to UriToBlob/BlobToUri?
to avoid confusion - folks may think "Text' means contents.

Reply:
Would be a breaking change

Reply:
Sorry I thought you wanted rename the UnaryOpKind